cli/src/base.ts

```diff
@@ -346,14 +346,25 @@ export abstract class BaseCommand<T extends typeof Command> extends Command {
     workspace: string,
     region: string,
     database: string,
-    options: { allowEmpty?: boolean; allowCreate?: boolean; title?: string } = {}
+    options: {
+      allowEmpty?: boolean;
+      allowCreate?: boolean;
+      title?: string;
+      useBranchIfExists?: { branch: string };
+    } = {}
   ): Promise<string> {
     const xata = await this.getXataClient();
     const { branches = [] } = await xata.api.branches.getBranchList({ workspace, region, database });
 
     const EMPTY_CHOICE = '$empty';
     const CREATE_CHOICE = '$create';
 
+    if (options.useBranchIfExists) {
+      if (branches.map(({ name }) => name).includes(options.useBranchIfExists.branch)) {
+        return options.useBranchIfExists.branch;
+      }
+    }
+
```

Comment:
Can be probably simplified

```suggestion
    if (branches.map(({ name }) => name).includes(options.defaultBranch)) {
      return options.useBranchIfExists.branch;
    }

```

Reply:
It's potentially undefined so I need to check it exists first. I can simplify it to this though:

```js
 if (options.defaultBranch && branches.map(({ name }) => name).includes(options.defaultBranch)) {
      return options.defaultBranch;
    }
```